api/resources/translations/messages-es.properties

```diff
@@ -298,72 +298,72 @@ admin.message.queue.scheduled = Programado
 admin.message.queue.status = Estado
 admin.message.queue.tab.due = Vence
 admin.message.queue.tab.muted.future = No se enviará
-admin.message.queue.tab.muted.past = 
-admin.message.queue.tab.scheduled = 
-admin.message.queue.updated =
-admin.message.queue.view.report = 
-admin.pagination.detail =
-admin.pagination.first = 
-admin.pagination.last =
-admin.pagination.next = 
-admin.pagination.prev =
-admin.targets.description = 
-after\ the = Después de
+admin.message.queue.tab.muted.past = No se envió
+admin.message.queue.tab.scheduled = Programado
+admin.message.queue.updated = Última actualización
+admin.message.queue.view.report = Ver informe
+admin.pagination.detail = Mostrando {{first}} - {{last}} de {{total}} elementos
+admin.pagination.first = Primero
+admin.pagination.last = Último
+admin.pagination.next = Siguiente
+admin.pagination.prev = Anterior
+admin.targets.description = Una meta de -1 mostrará un objetivo sin meta.
+after\ the = después del
 analytics.anc.active-pregnancies = Embarazos activos
-analytics.anc.delivery-locations = Sitios de partos reportados
+analytics.anc.delivery-locations = Ubicaciones de partos reportados
 analytics.anc.high-risk = Embarazos de alto riesgo
 analytics.anc.missed-appointments = Citas perdidas recientes
-analytics.anc.missing-reports = Informes de parto faltando
-analytics.anc.monthly-births = Partos mensuales
-analytics.anc.monthly-registrations = Embarazos mensuales registrados
-analytics.anc.total-births = Partos totales
+analytics.anc.missing-reports = Informes de nacimiento faltantes
```

Comment:
I´m now sure if I´m getting this right. What do you think of `'Informes faltantes de nacimiento'`